_ont/ont-sercomm-fg1000b-11.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sercomm FG1000B.11
+has_children: false
+layout: default
+parent: Sercomm
+---
+
+# Hardware Specifications
+
+|                 |                        |
+| --------------- | ---------------------- |
+| Vendor/Brand    | Sercomm                |
+| Model           | FG1000B.11             |
+| Chipset         | BCM68360_B1            |
+| Flash           | NAND 128 MB            |
+| RAM             | 256 MB                 |
+| CPU             | Broadcom B53 Dual Core |
+| CPU Clock       | 1500MHz                |
+| Bootloader      | CFE                    |
+| Load addr       | 0x0000000000080000     |
+| 2.5GBaseT       | Yes                    |
+| PHY Ethernet    | RTL8221B               |
+| Optics          | SC/APC                 |
+| IP address      | 192.168.100.1/24       |
+| Web Gui         | ✅, no login needed    |
+| SSH             | No                     |
+| Telnet          | No                     |
+| Serial          | No                     |
+| Serial baud     | 115200                 |
+| Serial encoding | 8-N-1                  |
+| Form Factor     | ONT                    |
+
+
+{% include image.html file="fg1000b-11_rear.jpg" alt="Sercomm FG1000B.11" caption="Sercomm FG1000B.11 rear" %}
+{% include image.html file="fg1000b-11_bottom.jpg" alt="Sercomm FG1000B.11 bottom" caption="Sercomm FG1000B.11 bottom" %}
+{% include image.html file="fg1000b-11_side1.jpg" alt="Sercomm FG1000B.11 side 1" caption="Sercomm FG1000B.11 side 1" %}
+{% include image.html file="fg1000b-11_side2.jpg" alt="Sercomm FG1000B.11 side 2" caption="Sercomm FG1000B.11 side 2" %}
+
+# Serial
+
+See picture side2 for the pin identification, use 112500 8-N-1
+The ONT seems only to display output of the ROM CFE and flash CFE, but don't allow interupting the boot...
+
+dump:
+
+{% include serial_dump.html file="fg1000b-11_boot_cfe.txt" alt="Sercomm FG1000B.11 CFE boot dump" %}
+
+## Getting/Setting ONU GPON PLOAM password
```

Comment:
please use the new layout, which has now been introduced on almost all pages (except f601)

Reply:
Not sur what you mean here I've re adjust heading 1 and 2 and the order ...